interface/patient_file/encounter/find_code_dynamic_ajax.php

```diff
@@ -0,0 +1,333 @@
+<?php
+// Copyright (C) 2015-2017 Rod Roark <rod@sunsetsystems.com>
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+
+$sanitize_all_escapes  = true;
+$fake_register_globals = false;
+
+require_once("../../globals.php");
+require_once("$srcdir/formdata.inc.php");
+require_once("$srcdir/formatting.inc.php");
+// require_once("$srcdir/jsonwrapper/jsonwrapper.php");
```

Comment:
remove above 3 lines since first 2 are autoloaded and the third is commented out:
https://github.com/openemr/openemr/blob/master/composer.json#L56-L62